include/mbedtls/sha256.h

```diff
@@ -253,11 +279,11 @@ MBEDTLS_DEPRECATED static inline void mbedtls_sha256(
 
 #undef MBEDTLS_DEPRECATED
 #endif /* !MBEDTLS_DEPRECATED_REMOVED */
-
+						
```

Comment:
Blocker: Please remove the tabs here.

Reply:
Removed. I think these were pre-existing, as on lines I did not change.